app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/MemoriesFragment.java

```diff
@@ -139,14 +139,20 @@ public void onClick(View v) {
         return view;
     }
 
+    @Override
+    public void onPause() {
+        super.onPause();
+        if(actionMode != null){
+        actionMode.finish();}
+    }
+
```

Comment:
It's clear that you didn't `ctrl+alt+l`. If you have a second, please still do that. If not, that's fine too.

Reply:
I ctrl+alt+l-ed